lib/presentation/widget/upload_file/upload_file_widget.dart

```diff
@@ -48,6 +51,7 @@ class _UploadFileWidgetState extends State<UploadFileWidget> {
   final uploadFileViewModel = getIt<UploadFileViewModel>();
   final imagePath = getIt<AppImagePaths>();
   final appNavigation = getIt<AppNavigation>();
+  final TextEditingController _typeAheadController = TextEditingController();
```

Comment:
I saw the cookbook of Flutter. I found that Flutter prefer a `TextEditingController` should be disposed after the holding widget is disposed. Plz consider https://flutter.dev/docs/cookbook/forms/text-field-changes

Reply:
can you describe more about that

Reply:
Ive read the code of dispose, it just free out added listeners. In this our case, we don't have any listener, but with the responsible of a person who did at first place, I think we need to take care when any listeners will be added in the future :D

Reply:
Agree, lets call dispose whenever use this.